inject-groovy/src/test/groovy/io/micronaut/inject/configuration/GroovyConfigurationMetadataBuilderSpec.groovy

```diff
@@ -1,94 +0,0 @@
-/*
- * Copyright 2017-2019 original authors
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- * http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-package io.micronaut.inject.configuration
-
-import groovy.json.JsonSlurper
-import io.micronaut.ast.transform.test.AbstractBeanDefinitionSpec
-import io.micronaut.ast.groovy.config.GroovyConfigurationMetadataBuilder
-import org.codehaus.groovy.ast.ClassNode
-/**
- * @author graemerocher
- * @since 1.0
- */
-class GroovyConfigurationMetadataBuilderSpec extends AbstractBeanDefinitionSpec {
```

Comment:
I have added a new test. It looks like the metadata for the Groovy properties wouldn't be generated because the writer is only present for Java annotation processing.